### DISCUSSION TITLE
CCPM helicopter

### DISCUSSION THREAD

**Discussion by and-sh** [iNavFlight/inav#10234] [Q&A]
Hi
![Screenshot from 2024-07-14 13-14-35](https://github.com/user-attachments/assets/8f54a845-8cfc-4dfb-830a-29259a77229e)

Is this correct mixer for CCPM flaybared heli (TT Raptor 30)? No 3D. 

roll -> cyclic Roll servo
pitch -> cyclic pitch servo
throttle -> collective pitch servo
Yaw -> Yaw servo

If I need to invert can I set weitght "-100" ?

Can I read somewhere else about such setup?

**Comment by Jetrell** [comment]
INAV doesn't provide conventional helicopter support, with cyclic and collective mixing.
This is where the support was up to last. https://github.com/iNavFlight/inav/pull/9271

**Comment by and-sh** [comment]
Thank you. So it is possible, but with some limitations.

**Reply by Jetrell** [reply]
To say possible. Would be stating it very loosely..
It requires the user to add their own advanced mixing. And still providing very limited performance.

**Comment by and-sh** [comment]
I understand correctly that the same thing happens with the I term at the start as in betaflight. Does it not accumulate until the throttle passes a certain value?

**Reply by Jetrell** [reply]
The setting is known as - `airmode_throttle_threshold`

**Comment by and-sh** [comment]
Thank you. 
Is there something like this for Inav?

[PID 4i2.pdf](https://github.com/user-attachments/files/16231395/PID.4i2.pdf)
